From 126531e01b186a377925cb462a85a438e5d345e5 Mon Sep 17 00:00:00 2001
From: Adrian Pop <pop.adrian61@gmail.com>
Date: Sun, 7 Mar 2021 11:58:14 +0000
Subject: [PATCH] Fix yocto build error.

---
 AAServer/include/InputChannelHandler.h | 1 +
 1 files changed, 1 insertions(+), 0 deletion(-)

diff --git a/AAServer/include/InputChannelHandler.h b/AAServer/include/InputChannelHandler.h
index 0b44d8f..93e9f88 100644
--- a/AAServer/include/InputChannelHandler.h
+++ b/AAServer/include/InputChannelHandler.h
@@ -4,6 +4,7 @@
 
 #include "ChannelHandler.h"
 #include <vector>
+#include <set>
 
 class InputChannelHandler : public ChannelHandler {
   void sendHandshakeRequest();
